lettuce/flows/__init__.py

```diff
@@ -8,11 +8,13 @@
 from lettuce.flows.poiseuille import PoiseuilleFlow2D
 from lettuce.flows.doublyshear import DoublyPeriodicShear2D
 from lettuce.flows.decayingturbulence import DecayingTurbulence
+from lettuce import D2Q9, D3Q27
```

Comment:
please `from lettuce.stencil`

if you do the general imports within the package, the order of imports in lettuce/__init__.py will matter